data/json/martialarts.json

```diff
@@ -33,7 +33,7 @@
         "name": "Aikido Stance",
         "description": "By disregarding offensive in favor of self-defense, you are better at protecting.\n\nBlocked damage reduced by 100% of Dexterity.",
         "unarmed_allowed": true,
-        "flat_bonuses": [ { "stat": "block", "scaling-stat": "dex", "scale": 1.0 } ]
+        "flat_bonuses": [ [ "stat": "block", "scaling-stat": "dex", "scale": 1.0 ] ]
```

Comment:
I am fairly sure this was correct before. Using square brackets around this part is not valid JSON syntax.

Reply:
This was based on an incorrect suggestion I made based on a reported error message.  The underlying issue was loading new-style json in an older game binary.